eslint-example/README.md

```diff
@@ -0,0 +1,10 @@
+# ESLint Example
+
+## What it shows
+
+A project configured to use the [Mozilla ESlint configuration](https://www.npmjs.com/package/eslint-plugin-mozilla). By using a linter on your extension source code you can be sure that you will only be writing JavaScript that is compatible with Firefox.
```

Comment:
oops, this opening sentence is no longer true so it will be confusing. I would suggest replacing it with this:

"This is an example of a WebExtension project configured for development with [eslint](http://eslint.org/)."